dissect/target/plugins/os/windows/generic.py

```diff
@@ -1,5 +1,5 @@
 from datetime import datetime
-from typing import Optional
+from typing import Iterator, Optional
```

Comment:
```suggestion
from typing import Iterator
```
Can you also make all other `Optional`'s a `| None`?